src/code/templates/fv-egg-game.js

```diff
@@ -991,6 +992,28 @@ export default class FVEggGame extends Component {
       onResetGametes();
     };
 
+    function getUnselectedChromosomesMap(chromosomes, selectedChromosomes) {
```

Comment:
Here's where I'm now building up a map of unselected chromosomes. In general, the code is much cleaner now, even though I think it might have been cleaner to null a full mapping instead of building it from nothing here